changelog/3899.bugfix.rst

```diff
@@ -0,0 +1 @@
+`sunpy.net.hek.HEKClient` was returning HTML and not JSON.
```

Comment:
Should it be `~sunpy.net.hek.HEKClient` ?

Reply:
It can be.